ext/date/tests/DateTime_verify.phpt

```diff
@@ -27,7 +27,7 @@ object(ReflectionClass)#%d (1) {
   string(8) "DateTime"
 }
 ..and get names of all its methods
-array(18) {
+array(19) {
```

Comment:
What's the point of this test? (I realise, it has nothing to do with the PR). I'd say we remove it as it just tests that Reflection works.